content/ko/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -150,8 +150,8 @@ _집합성 기준_ 요건은 _일치성 기준_ 요건과 조합해서 사용할
 
 LIST와 WATCH 작업은 쿼리 파라미터를 사용해서 반환되는 오브젝트 집합을 필터링하기 위해 레이블 셀렉터를 지정할 수 있다. 다음의 2가지 요건 모두 허용된다(URL 쿼리 문자열을 그대로 표기함).
 
-  * _불일치 기준_ 요건: `?labelSelector=environment%3Dproduction,tier%3Dfrontend`
```

Comment:
The original is correct. The characters `%`, `3` and `D` are valid Unicode grapheme clusters. The text inside the backticks is part of a URL-encoded string.

It's a similar story for the following line.